src/pages/iou/request/step/IOURequestStepDescription.js

```diff
@@ -79,7 +87,7 @@ function IOURequestStepDescription({
         }
 
         // Only update comment if it has changed
-        if (newComment === lodashGet(transaction, 'comment.comment', '')) {
+        if (newComment === currentDescription) {
             navigateBack();
             return;
         }
```

Comment:
Given that `currentDescription` now considers the `splitDraftTranscation` we can move this condition to be the first